base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1320,6 +1328,118 @@ where
         .await
     }
 
+    /// Creates a transaction to burn some Tari
+    /// # Arguments
+    /// 'amount': The amount of Tari to send to the recipient
+    /// 'fee_per_gram': The amount of fee per transaction gram to be included in transaction
+    pub async fn burn_tari(
+        &mut self,
+        amount: MicroTari,
+        fee_per_gram: MicroTari,
+        message: String,
+        transaction_broadcast_join_handles: &mut FuturesUnordered<
+            JoinHandle<Result<TxId, TransactionServiceProtocolError<TxId>>>,
+        >,
+    ) -> Result<TxId, TransactionServiceError> {
+        let tx_id = TxId::new_random();
+        let output_features = OutputFeatures::create_burned_output();
```

Comment:
naming nit: consistency with `create_coinbase` and `KernelFeatures::create_burned` no output is being created.
```suggestion
        let output_features = OutputFeatures::create_burned();
```